packages/client/hmi-client/src/views/TA2Playground.vue

```diff
@@ -73,6 +74,10 @@ interface NodeData {
 interface EdgeData {
 	val: number;
 }
+enum NodeType {
+	species = 'S',
```

Comment:
Nitpick: it's a convention to capitalize enum constants, like
```suggestion
	Species = 'S',
```